solr/core/src/java/org/apache/solr/util/SolrCLI.java

```diff
@@ -2845,8 +2845,42 @@ protected void runExample(CommandLine cli, String exampleName) throws Exception
           echo("exampledocs directory not found, skipping indexing step for the techproducts example");
         }
       }
+      else if ("films".equals(exampleName) && !alreadyExists) {
+        echo("Adding name and initial_release_data fields to films schema \"_default\"");
 
-      echo("\nSolr "+exampleName+" example launched successfully. Direct your Web browser to "+solrUrl+" to visit the Solr Admin UI");
+        HttpSolrClient solrClient = new HttpSolrClient.Builder(solrUrl).build();
+        try {
+          SolrCLI.postJsonToSolr(solrClient, "/" + collectionName + "/schema", "{\n" +
```

Comment:
> Example of using the _default configset and adding fields dynamically.

That makes me think that films is just using the dynamic fields available in _default, but here you are adding a few fields. Maybe we convert it to only use dynamic fields, or change the description to be a little clearer.

Reply:
Took another stab at this!   If you think using _default is just a bad idea/not helpful, would you mind opening another JIRA and I'll do the migration there ;-)????

Reply:
Reading through https://github.com/apache/solr/tree/main/solr/example/films, the whole example makes a lot of assumptions that you are using _default.   If we get rid of schema autoguessing, then this whole example provides much less value.    (Oh, and despite it being an example of the `useParams` parameter according to ref guide, I don't actually see it mentioned or used!  sigh)

Reply:
I ended up digging into the README for the films example and created https://issues.apache.org/jira/browse/SOLR-15834